react-native/react/store/configure-store.native.desktop.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+/* @flow */
+
+import { createStore, compose } from 'redux'
+import { devTools } from 'redux-devtools'
+
+export default function configureStoreNative (rootReducer, initialState, createStoreWithMiddleware) {
+  const nativeCreateStoreWithMiddleware = compose(
+    createStoreWithMiddleware,
+    devTools()
+  )(createStore)
+
+  return nativeCreateStoreWithMiddleware(rootReducer, initialState)
+}
```

Comment:
My thinking was the platforms would have whatever custom stuff they wanted and not just a specific middleware injection so I'm passing all the common pieces to them to compose / do whatever and make the thing